### PR TITLE
Deduplicate content from 'Eliminate friction' section

### PR DESCRIPTION
**Pull request by mfix22**
IMO, the content under "Eliminate Friction from Using and Contributing to Open Source" is duplicated in other sections, most strongly in the InnerSource one. This PR distributes that content into the other sections.

I also removed "Continuously improve processes and tools to reduce learning curve and manual effort required" since it felt redundant—we already have 2 other sections on improving processes and policies, and other bullets on making contributing better or easier. 🙂 

What do you think @tsteenbe @anajsana?

### CODE REVIEW

**Comment by tsteenbe** [comment]
@mfix22 Not in favor of this PR - "Eliminate Friction from Using and Contributing to Open Source" section is not about InnerSource at all. It's about making it easier for development organization to consume open source and contribute back upstream / start new OSS projects. InnerSource is a a practices focused within an organization.

Keep also in mind that mind map is meant to shows "what OSPO could do" which is why there is some duplication of items between points. Well aware of several OSPO that are not responsible for InnerSource but are tasked to make using and contributing to Open Source easier.

File: ospo-mindmap/Content/ospomindmap.md
Comment (tsteenbe) [inline ospo-mindmap/Content/ospomindmap.md:309]: I do see how "Help navigate internal politics or policies" could be seen as sub point of "Improve collaboration across the organization" / InnerSource section. If we adding this as a subpoint I would prefer if we add another example else it look weird to me in the rendered mind map

**Review by tsteenbe — CHANGES_REQUESTED** [review]

**Comment by mfix22** [comment]
Makes sense @tsteenbe ! I just found this section to be overly redundant, when we have 2 other section specifically on using and contributing to open source. 

I attempted to reorganize this again. What do you think about this latest iteration? 

If you still disagree that this is worth doing, I will close this PR 🙂

**Comment by mfix22** [comment]
@tsteenbe any thoughts on the latest iteration here? If you still are not a fan, I'll close 🙂

**Comment by mfix22** [comment]
@tsteenbe just wanted to get your approval on the above before merging 🙂

File: ospo-mindmap/Content/ospomindmap.md
Comment (tsteenbe) [inline ospo-mindmap/Content/ospomindmap.md:152]: "Help navigate internal politics or policies" is not only a compliance topic but also a contributing to Open Source  - what if an engineer/product owner wants to open-source a big closed source application with several internal dependencies and approvals from multiple directories and above. OSPO is the team that can help with these kind of tasks that require existing relationships to higher management

File: ospo-mindmap/Content/ospomindmap.md
Comment (tsteenbe) [inline ospo-mindmap/Content/ospomindmap.md:91]: If we start elaborating the execution part of the strategy then "Maintain relationships..." should be a sub bullet under Open Source community, contributions and leadership

**Review by tsteenbe — CHANGES_REQUESTED** [review]
I still prefer not to remove "Eliminate Friction from Using and Contributing to Open Source" - see value in it as a standalone responsibility.

**Comment by mfix22** [comment]
Going to close this for now. Thanks for reviewing @tsteenbe.

**Pull request closed without merging**
